pkg/reaper/README.md

```diff
@@ -133,6 +138,22 @@ Flags:
       --soft-reap                          Will not terminate nodes with running pods (default true)
 ```
 
+#### Locking node reaper
+
+It is possible to coordinate multiple instances of `node-reaper` to avoid reaping more than one control plane node at a time. This may be desirable in some situations, notably the one
+described in [kops#13686](https://github.com/kubernetes/kops/issues/13686).
+
+The locking mechanism utilizes a DynamoDB table accessible to all instances of `node-reaper`. Expected table attribute is
+
+```hcl
+attribute {
```

Comment:
Missing `ClusterID`

Reply:
Turned out to be not necessary. I tried with `ClusterID` and `PutItem` was adding items instead of failing when there's one already. Maybe could've added a conditional expression but not sure there's any practical difference.

Reply:
In that case, the sentence below should be updated?

> where `LockType` is the hash key, and `ClusterID` is the range key.